bubble-wrap.yaml

```diff
@@ -2692,7 +2695,6 @@ layers:
         # building footprints, pre-extrusion
         footprints:
             filter:
-                $zoom: { max: 18 }
```

Comment:
K, I am going to this change out. @nvkelso can you prioritize https://github.com/tangrams/bubble-wrap/issues/200? We plan to send this over for tizen dev asap.

Thanks